src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const statesArr = [];
```

Comment:
so many possible valid names
```suggestion
  const stateClones = [];
```
or
```suggestion
  const stateHistory = [];
```
or
```suggestion
  const stateVersions = [];
```